src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx

```diff
@@ -148,19 +150,18 @@ export default class WorkloadPodLogs extends React.Component<WorkloadPodLogsProp
     if (this.props.pods.length < 1) {
       this.state = {
         duration: retrieveDuration() || 600,
+        hideLogValue: '',
```

Comment:
Aha! I see you like things alphabetical :)
I was wondering why there we so many things changed but I guess some of it is just lines moving around.

Reply:
+1, I am a little OCD on the alphabetizing